proposals/0002-root-signature-in-clang.md

```diff
@@ -308,14 +309,21 @@ subsequent tokens. Additionally, all keywords and enums are case-insensitive.
 
     STATIC_BORDER_COLOR = 'STATIC_BORDER_COLOR_TRANSPARENT_BLACK' |
                           'STATIC_BORDER_COLOR_OPAQUE_BLACK' |
-                          'STATIC_BORDER_COLOR_OPAQUE_WHITE' ;
+                          'STATIC_BORDER_COLOR_OPAQUE_WHITE' |
+                          'STATIC_BORDER_COLOR_OPAQUE_BLACK_UINT' |
+                          'STATIC_BORDER_COLOR_OPAQUE_WHITE_UINT' ;
+
+    STATIC_SAMPLER_FLAGS = 'SAMPLER_FLAG_UINT_BORDER_COLOR' |
+                           'SAMPLER_FLAG_NON_NORMALIZED_COORDINATES';
 ```
 
 ### Root Signature Versioning 
 
-Currently, DirectX supports two "versions" of root signatures: 1.0 and 1.1. 
-Version 1.1 includes additional flags for descriptor ranges and root descriptors. 
+Currently, DirectX supports three "versions" of root signatures: 1.0, 1.1 and 1.2. 
+Version 1.1 includes additional flags for descriptor ranges and root descriptors.
 See the [DirectX Documentation][root_signature_versions_doc] for full details.
+Version 1.2 includes additions flags for static samplers. This changes were documented
+in the [Vulkan Specification][root_signature_version12_doc] for root signatures.
```

Comment:
This isn't the "Vulkan Specification" - this is the spec for VulkanOn12 (that is, the implementation of Vulkan on D3D12)